app/DuesTransaction.php

```diff
@@ -37,6 +48,21 @@ public function user()
         return $this->belongsTo('App\User');
     }
 
+
+    /**
+     * Get the status flag for the Transaction.
+     *
+     * @return bool
+     */
+    public function getStatusAttribute()
+    {
+        if ($this->payment->first() != null) {
+            return "Paid";
```

Comment:
I like the idea of using `count()`. The capitalization is so that I don't have to write a render function on the frontend, but that's a bad justification.